action/lint.go

```diff
@@ -0,0 +1,160 @@
+package action
+
+import (
+	"github.com/google/go-github/github"
+	"github.com/helm/helm/chart"
+	"github.com/helm/helm/log"
+	"github.com/helm/helm/manifest"
+	"gopkg.in/yaml.v2"
+	"io"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+)
```

Comment:
I've been doing alphabetical order but I would be into what you're suggesting. @technosophos @adamreese any thoughts?